configure.ac

```diff
@@ -39,7 +39,7 @@ AC_PATH_TOOL(RANLIB, ranlib)
 AC_PATH_TOOL(STRIP, strip)
 
 AM_PROG_CC_C_O
-AC_PROG_CC_C89
+AC_PROG_CC
```

Comment:
I think the `AM_PROG_CC_C_O` should be below `AC_PROG_CC`.

In fact, this has always been wrong but never caused issues. And we should actually drop that one, too... It's obsolete since Automake 1.14 (2013), see https://www.gnu.org/software/automake/manual/html_node/Public-Macros.html

Reply:
Good catch, I'll remove that one too.